simulator/simple_planning_simulator/src/simple_planning_simulator_core.cpp

```diff
@@ -93,19 +96,19 @@ Simulator::Simulator(const std::string & node_name, const rclcpp::NodeOptions &
   {
     auto vehicle_model_type_str = declare_parameter("vehicle_model_type", "IDEAL_STEER");
     RCLCPP_INFO(get_logger(), "vehicle_model_type = %s", vehicle_model_type_str.c_str());
-    auto tread_length = declare_parameter("tread_length", 2.0);
-    auto angvel_lim = declare_parameter("angvel_lim", 3.0);
+    // auto tread_length = declare_parameter("tread_length", 2.0);
```

Comment:
Again, what's the reason for commenting them out?

Reply:
deleted!